ra_aid/agent_utils.py

```
@@ -569,3 +569,25 @@ def run_agent_with_retry(agent, prompt: str, config: dict) -> Optional[str]:
+
+class AgentSupervisor:
+    def __init__(self, model=None, max_turns=10):
+        self.model = model or _global_memory.get('config', {}).get('model')
+        self.max_turns = max_turns
+        self.checklist = []
+
+    def load_checklist(self, checklist_path: str):
+        with open(checklist_path, 'r') as file:
+            self.checklist = file.readlines()
+
+    def save_checklist(self, checklist_path: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `save_checklist`

Add return type annotation: `None`

(ANN201)